Formula/g/ghc.rb

```diff
@@ -147,7 +119,7 @@ def install
       -j#{ENV.make_jobs}
       --prefix=#{prefix}
       --flavour=release
-      --docs=no-sphinx-pdfs
+      --docs=no-sphinx
```

Comment:
Does this mean no more manpages? Is it broken?

You may want to check the "sphinx-doc 7" patch in `ghc@9.2` (the sphinx-doc 6 one should already be in included in the release).

Reply:
I will add it back with the patches